host-endpoints-externally/setup-accountrecovery-endpoint.sh

```diff
@@ -104,6 +104,7 @@ cp $IS_HOME/lib/runtimes/cxf3/org.wso2.carbon.identity.mgt.endpoint.util-*.jar $
 cp $IS_HOME/lib/runtimes/cxf3/jersey-client-*.jar $WEB_APP_LIB
 cp $IS_HOME/lib/runtimes/cxf3/jersey-core-*.jar $WEB_APP_LIB
 cp $IS_HOME/lib/runtimes/cxf3/jersey-multipart-*.jar $WEB_APP_LIB
+cp $IS_HOME/lib/runtimes/cxf3/org.wso2.carbon.identity.application.authentication.endpoint.util-*.jar $WEB_APP_LIB
```

Comment:
shouldn't this be `org.wso2.carbon.identity.application.authentication.endpoint.util_` instead of `org.wso2.carbon.identity.application.authentication.endpoint.util-`?

Reply:
As it is in `lib/runtimes/cxf3` folder, the libraries are defined in that format. In the plugins folder the libraries will be in the mentioned format